escrow/free_call_api.go

```diff
@@ -17,6 +17,13 @@ type FreeCallPayment struct {
 	CurrentBlockNumber *big.Int
 	// Signature passed
 	Signature []byte
+	//Group ID
+	GroupId string
+	//Auth Token Passed
+	AuthToken []byte
+	//Date on when the token was issued in block number
+	AuthTokenIssueDateBlockNumber *big.Int
```

Comment:
BTW, why not pass expiration date instead? 
1. It doesn't require some expiration date hardcode on the daemon side. 
2. It makes it clear for client when token is expected to be expired.